source/qml/SettingsPage.qml

```diff
@@ -956,7 +994,7 @@ ScrollView {
 
                 Switch{
                     id: rotatorSwitch
-                    enabled: (predicterController.tracking && rotatorCOMCombo.count > 0) || rotatorSwitch.checked
+                    enabled: (predicterController.tracking && rotatorCOMCombo.count > 0)
```

Comment:
Having `... ||  rotatorSwitch.checked` added here allowed the user to turn off the rotator control even if tracking has been disabled while rotator was enabled. I think we should not lose that ability.

Reply:
This does not disable the rotator when tracking is turned off, it only disables the switch itself. In order to do that, you would have to add some code to `trackingSwitch`'s `onCheckedChanged` (i.e. set `rotatorSwtich`'s `checked` to `false`)

Reply:
My bad, the `onCheckedChanged` logic I talked about is actually alread in place as `onEnabledChanged` a few lines below.